### ISSUE TITLE
WebComponent styles overriding my application bootstrap styles completely

### ISSUE THREAD

**Issue by VenkatGarlapati** [stoplightio/elements#1792]
## Describe the bug
When i am trying to integrate the stoplight elements component with my application it is overriding my existing styles completely. 

## To Reproduce

Integrate elements webcomponent with existing bootstrap styled application

## Expected behavior

Elements webcomponent styles should limit to the component it self.

## Additional context
https://unpkg.com/@stoplight/elements@7.3.5/styles.min.css
It seems like couple of styles written in the elements level globally, instead of that maybe we need to restrict it to the component level

```
blockquote,dd,dl,figure,h1,h2,h3,h4,h5,h6,hr,p,pre{margin:0}button{background-color:initial;background-image:none}
```

**Screenshots**
Before injecting the stoplight elemnents styles it is displaying in different way after injecting the web component with my application UI changed completly.

![image](https://user-images.githubusercontent.com/28777323/135242877-464e3478-6697-49da-a89c-ef0d9f91620f.png)

**Environment (remove any that are not applicable):**
 - Library version: [e.g. 7.3.5]
 - OS: Windows 10
 - Browser: Chrome
 - Platform: Web

**Comment by ajrice6713** [comment]
Seeing this behavior as well - attempted to implement the elements react component into my docusaurus site and its overriding all of my other site css. Would be awesome to have the elements css gated to the elements component so that this could be embedded anywhere easily.

**Comment by GideonShils** [comment]
Just bumping up this issue. Any chance we can get a hosted version of the styles that doesn't include the CSS resets?

**Comment by gvangool** [comment]
We've solved this by using [postcss-nested-import](https://github.com/eriklharper/postcss-nested-import) and [postcss-nested](https://github.com/postcss/postcss-nested).

Our new style sheet is (stored in `assets/css`):
```css
@import "@stoplight/mosaic/themes/default.css";

elements-api {
  @nested-import "../../node_modules/@stoplight/elements/styles.min.css";
}
```

PostCSS config:
```js
module.exports = {
  plugins: [
    require("postcss-nested-import"),
    require("postcss-nested"),
    require("autoprefixer"),
  ],
};
```

**Comment by hhtdev** [comment]
Any news on this ? I'm experiencing this issue and i'd like to know what's the recommended approach from the Stoplight team ?

**Comment by hotrush** [comment]
I agree with above, styles of embeddable elements must not affect global scope

**Comment by hotrush** [comment]
BTW, workaround from @gvangool works, but looks like `postcss-nested-import` is not needed anymore

```
module.exports = {
  plugins: {
    'postcss-nested': {},
    '@tailwindcss/postcss': {},
  },
};
```
and
```
@import "@stoplight/mosaic/themes/default.css";

elements-api {
  @import "../../node_modules/@stoplight/elements/styles.min.css";
}
```
works to me, but it adds almost 400kb of css...

**Comment by sgerace** [comment]
We're seeing the same thing with things like stoplight.css applying global rules such as:
```
table {
    border-collapse: collapse;
}
```

![Image](https://github.com/user-attachments/assets/521cd091-c107-4114-82eb-ad3bbdc4ccf7)

Which is causing some issues with some of our table layout. Any updates on whether this can be prioritized?

**Comment by thaoms** [comment]
Same in our Next.js app. 
When embedding it completely breaks our other css. How this is still a thing is beyond me.
